### DISCUSSION TITLE
Improve PR template

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#4956] [Ideas]
https://github.com/ScratchAddons/website-v2/pull/219#issuecomment-1203164488

If PR reviewers might need to fill out a checklist at some point, I think PR creators should too. Also, since all PRs require multiple reviews it might be a good idea to add the needs review label to the template.

**Comment by Samq64** [comment]
Here's kind of what I was thinking of:

```md
<!-- Which issue(s) does this pull request fix or resolve? -->

Resolves #

### Changes

<!-- Please describe the changes you've made -->

### Reason for changes

<!-- Why should these changes be made? -->

### Tests

[ ] Tested on a Chromium browser
[ ] Tested on Firefox

For new addons:
[ ] All settings and presets tested
[ ] Dynamic enable/disable works
[ ] Tested with all addons enabled

<!-- Include any notes about the tests here, including known issues or untested changes. -->

### Screenshots

<-- Add screenshots if this pull request adds a new addon or visual changes. -->
``

**Comment by DNin01** [comment]
Are we requiring users to test their addons in both a Chromium browser and Firefox or in at least one of them?

**Reply by WorldLanguages** [reply]
Related: https://github.com/ScratchAddons/website-v2/pull/219

**Reply by DNin01** [reply]
I suppose [this quality guideline](https://github.com/ScratchAddons/website-v2/blob/3b3c6693a8ddfd9a848a52836b03f0dedb0c9ccb/content/docs/reference/quality-guidelines.md?plain=1#L10) answers that with a both.

**Reply by mxmou** [reply]
> I suppose [this quality guideline](https://github.com/ScratchAddons/website-v2/blob/3b3c6693a8ddfd9a848a52836b03f0dedb0c9ccb/content/docs/reference/quality-guidelines.md?plain=1#L10) answers that with a both.

That doesn't say anything about testing, it just says the addon can only use APIs supported by both browsers. See https://github.com/ScratchAddons/website-v2/pull/219#discussion_r985843522

**Reply by DNin01** [reply]
> > I suppose [this quality guideline](https://github.com/ScratchAddons/website-v2/blob/3b3c6693a8ddfd9a848a52836b03f0dedb0c9ccb/content/docs/reference/quality-guidelines.md?plain=1#L10) answers that with a both.
> 
> 
> 
> That doesn't say anything about testing, it just says the addon can only use APIs supported by both browsers. See https://github.com/ScratchAddons/website-v2/pull/219#discussion_r985843522

Further down, it says, "Every PR should be tested on a Chromium-based browser (Chrome, Edge, Brave, or another) **and** Firefox."

**Reply by DNin01** [reply]
Okay, maybe this will turn out to be _either_ browser, or at least a Chromium-based browser - that seems to be the way the conversation is asking us to go.

**Reply by BroJac5246** [reply]
> Okay, maybe this will turn out to be _either_ browser, or at least a Chromium-based browser - that seems to be the way the conversation is asking us to go.

Some people can only test on one of the browsers, so it realistically _has_ to be that way.
